docs/how_to_install_collector.md

```diff
@@ -0,0 +1,50 @@
+How to install collector command
+=====================================
+
+The collector command can run in setup-hw container and bare metal linux server. 
+in this memo describe how to install bare metal server.
+
+## 1.Clone code from github repository
+
+```
+$ git clone https://github.com/cybozu-go/setup-hw
+```
+
+## 2.Build & install collector command
+
+```
+$ cd setup-hw
+$ make install
+```
+
+## 3.Setup config
+
+Put a bmc-user.json file in /etc/neco/ that must have "support" user due to use the collector command.
+
+```
+{
+  "support": {
+    "password": {
+      "raw": "cybozu"
+    }
+  }
+}
+```
+
+Put a bmc-address.json file in /etc/neco. 
+
+```
+{"ipv4":{"address":"10.210.152.197","netmask":"255.255.252.0","maskbits":22,"gateway":"10.210.152.198"}}
```

Comment:
ditto

`192.0.2.0/24` is a reserved IP address block for documentation and examples.

```suggestion
{
  "ipv4": {
    "address": "192.0.2.3",
    "netmask": "255.255.255.0",
    "gateway": "192.0.2.1"
  }
}
```